doc/UsersGuide/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -90,75 +162,60 @@ On most Level 1 systems, a container named ``ubuntu22.04-intel-ue-1.6.0-srw-dev.
 .. note::
    * The NOAA Cloud containers are accessible only to those with EPIC resources. 
 
-Users can simply set an environment variable to point to the container: 
+It is practical to set an environment variable to point to the container: 
 
 .. code-block:: console
 
    export img=/path/to/ubuntu22.04-intel-ue-1.6.0-srw-dev.img
 
-Users may convert the container ``.img`` file to a writable sandbox:
+Convert the read-only image in a shared location to a writable sandbox in a user's space:
```

Comment:
This is an optional step and the conditional "may" should still be kept.

Reply:
noted! changing it to the following:

`Users may convert the read-only image in a shared location to a writable sandbox in user's space:`

Reply:
Updated in the recent commit.